manim/utils/manim_banner.py

```diff
@@ -0,0 +1,107 @@
+"""Manim Banner utilities."""
+
+__all__ = ["ManimBanner"]
+
+from ..constants import LEFT, UP, RIGHT, DOWN, ORIGIN
+from ..animation.composition import Succession, AnimationGroup
+from ..animation.transform import ApplyMethod
+from ..mobject.geometry import Circle, Square, Triangle
+from ..mobject.svg.tex_mobject import Tex, MathTex
+from ..mobject.types.vectorized_mobject import VGroup
+from ..utils.tex_templates import TexFontTemplates
+
+
+class ManimBanner(VGroup):
+    r"""Convenience class representing Manim's banner.
+
+    Can be animated using the costum methods.
+
+    Examples
+    --------
+
+    .. manim:: BannerExample
+        from manim.utils.manim_banner import ManimBanner
+
+        class BannerExample(Scene):
+            def construct(self):
+                banner = ManimBanner().scale(0.25).to_corner(DR)
+                self.add(banner)
+                self.wait()
+                self.play(banner.expand())
+                self.wait()
+
+    """
+
+    def __init__(self, dark_theme=True):
+        VGroup.__init__(self)
+
+        logo_green = "#81b29a"
+        logo_blue = "#454866"
+        logo_red = "#e07a5f"
+        m_height_over_anim_height = 0.75748
+
+        self.font_color = "#ece6e2" if dark_theme else "#343434"
+        self.scale_factor = 1
+
+        ds_m = MathTex(r"\mathbb{M}").scale(7).set_color(self.font_color)
+        ds_m.shift(2.25 * LEFT + 1.5 * UP)
+        self.M = ds_m
+
+        circle = Circle(color=logo_green, fill_opacity=1).shift(LEFT)
+        self.circle = circle
+        square = Square(color=logo_blue, fill_opacity=1).shift(UP)
+        self.square = square
+        triangle = Triangle(color=logo_red, fill_opacity=1).shift(RIGHT)
+        self.triangle = triangle
+        self.add(self.triangle, self.square, self.circle, self.M)
+        self.move_to(ORIGIN)
```

Comment:
```suggestion
        self.M = MathTex(r"\mathbb{M}").scale(7).set_color(self.font_color)
        self.M.shift(2.25 * LEFT + 1.5 * UP)
        self.circle = Circle(color=logo_green, fill_opacity=1).shift(LEFT)
        self.square = Square(color=logo_blue, fill_opacity=1).shift(UP)
        self.triangle = Triangle(color=logo_red, fill_opacity=1).shift(RIGHT)
        self.add(self.triangle, self.square, self.circle, self.M)
        self.move_to(ORIGIN)
```

Reply:
why not like this?

Reply:
+1

Reply:
Yes, thank you: this version really used a lot of unnecessary local variables. I've rewritten and updated it.